### ISSUE TITLE
Wrong default sysconfdir

### ISSUE THREAD

**Issue by ryandesign** [squash/sudosh2#31]
`./configure --prefix=/opt/local` output includes:

```
  Installation prefix......: /opt/local
  Man directory............: /opt/local/share/man
  sysconfdir...............: /etc
```

despite the fact that the help output says the default should be /opt/local/etc:

```
./configure --help | grep sysconfdir
  --sysconfdir=DIR        read-only single-machine data [PREFIX/etc]
```

configure.in clearly contradicts my desires intentionally:

```
# Override default
test "$sysconfdir" = '${prefix}/etc' && sysconfdir='/etc'
```

I would recommend removing those lines.

**Comment by ryandesign** [comment]
Looks like more changes would be needed than just removing those lines. In addition, you would have to `eval` `sysconfdir` to expand the reference to `${prefix}` -- not just for display in configure output but also for the `#define SYSCONFDIR` in config.h.

**Comment by squash** [comment]
Fixed in cccf70d
Thanks!
